service/create/service.go

```diff
@@ -196,7 +226,7 @@ func (s *Service) Boot() {
 	})
 }
 
-func (s *Service) runMachines(spec awstpr.Spec, clusterName, prefix string) error {
+func (s *Service) runMachines(awsSession *awssession.Session, ec2Client *ec2.EC2, spec awstpr.Spec, tlsAssets *cloudconfig.CompactTLSAssets, clusterName, prefix string) error {
```

Comment:
The interface of the method is not that nice. It now takes 9 arguments. There is no technical issue with this, but it kind of hurts me as a human.